documentation/website/core/Footer.js

```diff
@@ -101,8 +101,6 @@ class Footer extends React.Component {
                             </a>
                         </div>
                     </section>
-
-                    <section className="copyright">{this.props.config.copyright}<p style={{marginBottom:'0px', marginLeft:'20px', marginRight:'20px'}}>Disclaimer: Apache StreamPipes (incubating) is an effort undergoing incubation at The Apache Software Foundation (ASF), sponsored by the Apache Incubator. Incubation is required of all newly accepted projects until a further review indicates that the infrastructure, communications, and decision making process have stabilized in a manner consistent with other successful ASF projects. While incubation status is not necessarily a reflection of the completeness or stability of the code, it does indicate that the project has yet to be fully endorsed by the ASF.</p></section>
```

Comment:
We could add here instead of the Incubator disclaimer:
Copyright © 2022 The Apache Software Foundation.
Apache and the Apache feather logo are trademarks of The Apache Software Foundation.

Reply:
Copyright start should be 2019

Reply:
Soo, `Copyright © 2022 ` or `Copyright © 2019-2022`?

Reply:
I think Copyright (c) 2019 -2022 should be right

Reply:
I guess that's already part of the footer or am I mistaken?
https://github.com/apache/streampipes-website/blob/STREAMPIPES-639/website/partials/_footer.ejs#L4

Reply:
I also added it to the documentation footer in addition to the website.